package.json

```diff
@@ -5,18 +5,31 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
     "@material-ui/core": "^3.4.0",
+    "@material-ui/styles": "^3.0.0-alpha.3",
```

Comment:
If I remember correctly, Material UI styles are part of core i.e. `import { withStyles } from '@material-ui/core/styles';`. Is there any reason we're adding `@material-ui/styles` dependency here?